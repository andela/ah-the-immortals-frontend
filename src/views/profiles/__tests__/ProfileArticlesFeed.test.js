import {mount} from 'enzyme';
import React from 'react';
import ArticlesFeed from '../ProfileArticlesFeed';


describe('Test profile article feed', () => {
  let props = [];
  const wrapper = mount(
    <ArticlesFeed articles={props} />
  );
  it('test article feed ', () => {
    expect(wrapper).toMatchSnapshot();
  });
  it('test map function', () => {
    wrapper.find('PostFeed').find('PostItem').map(node => node.text());
    expect(wrapper).not.toBe(null);
  });
});
