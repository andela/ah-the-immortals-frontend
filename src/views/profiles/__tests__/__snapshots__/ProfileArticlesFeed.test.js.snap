// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test profile article feed test article feed  1`] = `ReactWrapper {}`;
